lib/resources/TransferReversals.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var StripeResource = require('../StripeResource');
+
+/**
+ * TransferReversals is a unique resource in that, upon instantiation,
```

Comment:
Yes, I believe that's what it means.

The comment is copied from `ChargeRefunds`: https://github.com/stripe/stripe-node/blob/master/lib/resources/ChargeRefunds.js